reposcan/database/package_store.py

```diff
@@ -80,7 +80,6 @@ def _populate_package_names(self, unique_names):
         cur = self.conn.cursor()
         self.logger.debug("Unique package names in repository: %d", len(unique_names))
         to_import = []
-        self.package_name_map = self._prepare_table_map(["name"], "package_name")
```

Comment:
can't simply remove it, it's here intentionally - see c6e89e3d7727ed81409e01c3b4619bad4de3fd46

it's not optimal and the issue is tracked here - https://github.com/RedHatInsights/vmaas/issues/494

Reply:
Sorry. I thought it's redundant and tests passed after removing. I see it's good that review is done by you.

Reply:
actually I think it may be deleted, it's beacuse of modules_store.py inserting to the package_name table but whole modules_store.py is unused

Reply:
:smile: , ok, cool